### ISSUE TITLE
Instructor's name differs in two sections

### ISSUE THREAD

**Issue by tyrinwu** [williamyeny/duke-registration-enhancer#26]
See the image below. The _instructor_ on the top right corner says, Jennifer Ansley. But in the description sections, it suggests Schonberg. One may be correct if they are consistent... 

Currently, the project uses the instructor's from the description part.

<img src="https://user-images.githubusercontent.com/11923768/29251462-1fd88c1c-8023-11e7-801b-fc2d6979c9b6.png" width="600">

**Comment by williamyeny** [comment]
Thanks for the heads up. We parse all data from the description, so the instructor may be out of date if it is copied from the previous semester. Perhaps it will be better to separate the description and instructor for the most up-to-date instructor data.

**Comment by williamyeny** [comment]
In general, the "Instructor" section lists all possible instructors rather than the particular instructor that semester.

For example: CS201 description says it has instructors Astrachan, Forbes or Rodger. It does not specify which one for this particular semester. 

Therefore, an "instructor" button seems necessary. However, I'm not sure if I can find the endpoint for finding instructor data.

**Comment by tyrinwu** [comment]
Yes, even in the meeting information section of CS 201, it gives two names, Jeffrey R Forbes, Owen L Astrachan. 

I am just thinking that in the instructor section in the meeting information section, it gives full names rather than just last names, which may make searching for professors' information more easily. In the description section, it just gives last names.

Because of the full name reason, I would vote for using the instructor's name from the meeting information section even though it still does not give who exactly teaches the course.

**Comment by williamyeny** [comment]
Great idea, I didn't even think about using the instructor names from the meeting sections!

Since the endpoint I'm currently calling parses the data for each section anyway, it should be trivial to retrieve instructors. The only caveat I foresee is if there's multiple sections -- I think we can just take the first section though, as that's what I'm currently doing.
